data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3360 @@
+	on decline
+		log `He Yi asked me to sell him my cloaking device, I refused.`
+
+
+
+ship "Shield Beetle Prototype" "Shield Beetle Prototype (Black)"
+	sprite "ship/hai marauder shield beetle"
+	thumbnail "thumbnail/hai marauder shield beetle"
+	add attributes
+		"hull" 5200
+		"spinal mount" 1
+		"cloak" .01
+		"cloaking energy" 10
+		"cloaking fuel" .5
+		"cloaking heat" 10
+		"engine capacity" 5
+		"outfit space" 2
+		"weapon space" -1
+	outfits
+		"Sunbeam" 4
+		"Flamethrower" 2
+		"Korath Detainer"
+		"Korath Grab-Strike"
+		"Korath Banisher"
+		"Bullfrog Anti-Missile" 3
+		"Large Heat Shunt"
+		"Systems Core (Large)" 2
+		"Outfits Expansion" 4
+		"Jump Drive"
+		"Fuel Processor"
+		"Wanderer Ramscoop"
+		`"Bufaer" Atomic Steering`
+		"Thruster (Planetary Class)"
+		"White Sun Reactor"
+		"Generator (Furnace Class)"
+		"Korath Repeater Rifle" 20
+		"Pulse Rifle" 50
+		"Dual Pulse Pistols" 10
+	gun 0 -7 "Korath Detainer"
+	gun -22 -102 "Sunbeam"
+	gun 22 -102 "Sunbeam"
+	gun -34 -68 "Sunbeam"
+	gun 34 -68 "Sunbeam"
+	gun -49 -38
+	gun 49 -38
+	gun -61 -14 "Flamethrower"
+	gun 61 -14 "Flamethrower"
+	turret -51 21.5 "Korath Grab-Strike"
+	turret 51 21.5 "Bullfrog Anti-Missile"
+	turret -56.5 75 "Korath Banisher"
+	turret 56.5 75 "Bullfrog Anti-Missile"
+	turret 0 85.5 "Bullfrog Anti-Missile"
+
+mission "Unfettered: Start Of The War"
+	name `Strength Of A Leader`
+	description `Prove your strength against Si-Ra and Hi-Ye in a battle. But do not kill them.`
+	source "Darkcloak"
+	to offer
+		has "event: Unfettered: shield beetle mass production"
+		has "event: Unfettered: new cooling system"
+		has "Unfettered: Special Cargo 2: done"
+		has "main plot completed"
+		has "language: Wanderer"
+	on offer
+		conversation
+			`You find Si-Ra and Hi-Ye waiting for you on the spaceport. Si-Ra starts speaking to you, "Are you ready for war, <last>?"`
+			choice
+				`	"Not yet, I will come back when I am."`
+					goto later
+				`	"I was born ready."`
+			`	"And may you die ready, as well." Hi-Ye comes closer and takes you aside to speak to you, privately. "I have managed to buy myself a ship with the money from our last business opportunity. It is much stronger than the old one! And as a warlord, he has a strong fleet as well. You may want to consider this seriously. Do not forget you can call the fight off, to continue it later once you are ready, by giving us a symbolic fee."`
+			choice
+				`	"You do not scare me."`
+					accept
+				`	"After thinking a bit... Maybe I should not take such risks, I will not participate in your war."`
+					decline
+			`	"That is good! Let us fight, on the same terms of honor as before. Take off when you will be ready, we will be waiting for you."`
+			label later
+			`	"Be sure to come back soon, war cannot wait forever."`
+				defer
+	
+	npc save disable
+		government "Hai (Unfettered Challenge)"
+		ship "Shield Beetle (Pulse)" "Intrepid II"
```

Comment:
I think it would be nice to see Si-Ra with some more prototype/special equipment here as well. Show he is advancing through the ranks to earn special access as well as the player.

Maybe not a Basilisk himself, but a variant with ionic blasters and/or the new coolers, etc.

Reply:
Well he is actually advancing trough the ranks, but at the start he really only has the basic thing, and he even has difficulties buying his new SB without the money you bring him, because he spends it all on drugs and gambling or stuff like that, and takes rash decisions.

He is probably an infiltrated member of the Mercenaries, that's why he was carrying the bars, because he needs money (if you don't steal them, smbd else will given his attitude).
He really isn't a warlord, and these new special weapons are only for them pretty much, given how hard to produce they are. 

And even then, ion doesn't fit his style, he wants firepower. A little tripulse turret would not be out of the question.

Reply:
Ooo, yea, if you already have specific plans to take him another direction that's even better, and TriPulse Shredders do seem much more his style. Just seemed kind of strange to see someone you've become so involved with that already has an in before you even get there still in just a plain old shield beetle next to both your own prototype and the warlord's :)